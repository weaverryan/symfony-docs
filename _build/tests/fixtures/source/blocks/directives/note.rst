
.. note::

    Sometimes we add notes. But not too often because they interrupt the flow.
