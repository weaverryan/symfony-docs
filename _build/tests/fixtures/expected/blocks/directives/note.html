<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-wrapper">
    <div class="admonition admonition-note">
        <p class="admonition-title">Note</p>
        <p>Sometimes we add notes. But not too often because they interrupt the flow.</p>
    </div>
</div>
</body>
</html>
